main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -1040,6 +1041,32 @@ static String guessBetterFormat(ImportingJob job, ArrayNode fileRecords, String
         return bestFormat;
     }
 
+    static String guessBetterFormat(File file, String fileEncoding, String bestFormat) {
```

Comment:
It looks like the bulk of the code in this method is duplicated from the method immediately above. 
Can things be refactored to reduce/eliminate the duplication?

Reply:
@tfmorris I have addressed this.